### PR TITLE
Feature/51852 Components refactor - atoms (part 1)

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: components/src/atoms/fieldset/Fieldset.stories.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const info = `
+  This component does not have any modifiers.
+`
+
+storiesOf('Atoms/Fieldset', module)
+  .add('Default', () => ({
+      components: { AFieldset },
+      template: `
+      <a-fieldset legend-text="Legend text">
+        <span>Simple text</span>
```
Comment (szafran89) [inline components/src/atoms/fieldset/Fieldset.stories.js]: Change slot to `Fieldset content`

File: components/src/atoms/heading/Heading.stories.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          :level="4"
+          tag="span"
+        >
+          Span level 4
+        </a-heading>
+    `
+    }),
+    { info }
+  )
+  .add('Page heading', () => ({
```
Comment (szafran89) [inline components/src/atoms/heading/Heading.stories.js]: Please remove page option from here and from styles

File: components/src/atoms/icon/Icon.html [hunk 3]
```diff
@@ -2,6 +2,8 @@
   class="icon"
   role="img"
 >
-  <title>{{ title }}</title>
+  <title>
+    <slot/>
+  </title>
```
Comment (szafran89) [inline components/src/atoms/icon/Icon.html]: Icon component is to rework, I'll describe it in next task

File: components/src/atoms/image/Image.stories.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { storiesOf } from '@storybook/vue'
```
Comment (szafran89) [inline components/src/atoms/image/Image.stories.js]: You can remove props from image

File: components/src/atoms/information/Information.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  <a-heading
+    :level="level"
+    class="information__title"
+  >
+    {{ title }}
+  </a-heading>
+  <a-paragraph class="information__text">
+    {{ text }}
+  </a-paragraph>
+</div>
```
Comment (szafran89) [inline components/src/atoms/information/Information.html]: This component should not be here, please move to previous place, in the future we will move it to cookbook

File: components/src/atoms/link/Link.js [hunk 6]
```diff
@@ -4,14 +4,6 @@ export default {
...
       required: true
     },
-    inverted: {
-      type: Boolean,
-      default: false
-    },
-    secondary: {
-      type: Boolean,
-      default: false
-    },
```
Comment (szafran89) [inline components/src/atoms/link/Link.js]: you can remove all props from link component and use in directly as attr on component 
`<a-link href="...`

File: components/src/atoms/loader/Loader.html [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <div
-  class="loader loader--visible"
+  class="loader"
   aria-live="polite"
   aria-label="Do not refresh the page"
```
Comment (szafran89) [inline components/src/atoms/loader/Loader.html]: we can't keep plain text in component, please move to story

File: components/src/atoms/message/Message.html [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<div
+  class="message"
+  role="alert"
+>
+  <slot />
+  <a-icon
+    :icon="icon"
+    class="message__icon"
+  />
+</div>
```
Comment (szafran89) [inline components/src/atoms/message/Message.html]: IMO we don't need this component at all, for now please move to previous place

File: components/src/atoms/preformatted/Preformatted.stories.js [hunk 9]
```diff
@@ -0,0 +1,31 @@
+import { storiesOf } from '@storybook/vue'
```
Comment (szafran89) [inline components/src/atoms/preformatted/Preformatted.stories.js]: Please remove props at all and set html as below
```
<pre class="preformatted">
  <slot />
</pre>

File: components/src/atoms/preformatted/Preformatted.stories.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    components: { APreformatted },
+    template: `
+      <a-preformatted>
+        Aliqua minim incididunt velit enim est et. Sint incididunt eiusmod exercitation culpa in enim ut minim cillum quis sint velit tempor consectetur. Excepteur nostrud ea labore aliqua qui fugiat et dolor reprehenderit reprehenderit laboris. Nisi fugiat do Lorem est mollit commodo aliquip. Consequat incididunt irure laboris enim eiusmod tempor ea esse eu fugiat tempor sint. Ex sunt ullamco laboris adipisicing deserunt quis sunt sint amet.
+      </a-preformatted>
+    `
+    }),
+    { info }
+  )
+  .add('Custom tag', () => ({
```
Comment (szafran89) [inline components/src/atoms/preformatted/Preformatted.stories.js]: remove `Custom tag` story

File: components/src/04-views/home/Home.html [hunk 11]
```diff
@@ -1,9 +1,9 @@
 <main class="home">
   <alpaca-banner
+    class="home__banner-content"
     :link="mainBanner.link"
     :image="mainBanner.image"
-    class="home__banner-content"
-    aria-label="Banner"
```
[the remark this answers is not in the document]
Comment (jszczech) [reply components/src/04-views/home/Home.html]: I didn't removed it, I grouped this props: objects with link, image and array with sources.

**Pull request merged**